### ISSUE TITLE
How to train a model to retain more texture details?

### ISSUE THREAD

**Issue by kelisiya** [csjliang/DASR#7]
I'm currently trying to train your model, but I found that when encountering leaves, lawns, sand grains, etc., the model recreates these scenes badly, how can I adjust the training loss to make the model support these scenes, or need to add more such scenes dataset?

**Comment by csjliang** [comment]
Hi, thanks for your attention to our work. The restoration of those regions of fine-grained details is challenging, especially when high perceptual quality is required and faces the unknown degradation in a real-world application. In other words, the generation capacity of the model might be limited in those regions if the input image is degraded severely. To enhance the performance, I think adding more training data with similar scenes and introducing local-based losses may be helpful. But most importantly, the degradation process in the training and testing should be carefully controlled if possible (when you apply the model to a specialized task), to keep consistency between training and testing.
